crates/wasmtime/src/config.rs

```diff
@@ -2701,6 +2703,76 @@ impl Config {
     pub fn gc_support(&mut self, enable: bool) -> &mut Self {
         self.wasm_feature(WasmFeatures::GC_TYPES, enable)
     }
+
+    /// Explicitly indicate or not whether the host is using a hardware float
+    /// ABI on x86 targets.
+    ///
+    /// This configuration option is only applicable on the
+    /// `x86_64-unknown-none` Rust target and has no affect on other host
```

Comment:
s/affect/effect/

It does likely have an *affect*, namely that it's mildly sad that it has to exist...

(with apologies for the spelling pedantry!)